frontend/src/store.js

```diff
@@ -12,9 +13,18 @@ const store = createStore({
     },
     mutations: {
         /**
-        * This will clear all user's data and unauthorize current session
+        * Signs user out of the application session.
+        *
+        * Wipes the stored user data and the session cookie.
+        *
+        * @throws {Error} if wiping out the session cookie was not successful
         */
-        logOut(state) {
+        async logOut(state) {
```

Comment:
If this is async, please move it to actions

Reply:
And remember to update use of this in WofHeader component.
(i mean, use 'dispatch' instead of 'commit')

Reply:
These vuex intricacies elude me unfortunately :( But I will read up on them

Reply:
Done